front/app/components/Form/Components/Controls/SentimentLinearScaleControl/utils.tsx

```diff
@@ -0,0 +1,85 @@
+import messages from '../messages';
+
+import Sentiment1Svg from './assets/sentiment_1.svg';
+import Sentiment2Svg from './assets/sentiment_2.svg';
+import Sentiment3Svg from './assets/sentiment_3.svg';
+import Sentiment4Svg from './assets/sentiment_4.svg';
+import Sentiment5Svg from './assets/sentiment_5.svg';
+
+export const getClassNameSentimentImage = (data, visualIndex) => {
```

Comment:
Probably safe to assume these will always be numbers
```suggestion
export const getClassNameSentimentImage = ([data, visualIndex](data: number, visualIndex: number)) => {
```